Pipfile

```diff
@@ -39,6 +39,8 @@ pytest-randomly = "==3.2.1"
 safety = "==1.8.5"
 tox = "==3.14.5"
 vyper = "==0.1.0b12"
+eth-tester = "==0.3.0b1"
```

Comment:
I'd remove this and the tests for the contract. Since the contract is already compiled here it does not make much sense to include this.